opentelemetry-proto/Cargo.toml

```diff
@@ -44,7 +44,7 @@ zpages = ["trace"]
 testing = ["opentelemetry/testing"]
 
 # add ons
-internal-logs = ["tracing"]
+internal-logs = []
```

Comment:
So there doesn't seem to be any mention of `internal-logs` anywhere in the crate. Why is there such a feature here?

Reply:
The "internal-logs" feature is used by internal logging macros defined in the "opentelemetry" crate. These macros expand at compile time, injecting the corresponding logging code into crates that use them and so this feature needs to be defined in those crates too. Ideally, it would be good if this can be avoided, while still allowing individual crates to independently enable or disable logging.

Reply:
@lalitb I know exactly what the feature does in the `opentelemetry` crate, but the fact that it has no dependencies here and is not referenced in code makes me think that it’s not needed in this crate.

Reply:
I think the configuration is/was actually incorrect, it should include the reference to the identically named feature in the `opentelemetry` crate.

Reply:
@bantonsson i believe it is being referenced indirectly [here](https://github.com/open-telemetry/opentelemetry-rust/blob/99cb67d19ca6ad4df9a87cb31473a130827740c9/opentelemetry-proto/src/transform/metrics.rs#L10)

Reply:
Indeed it is. Then the omission of `internal-logs = ["opentelemetry/internal-logs"]`  wasn't caught by CI before since we didn't test the features separately.